analytics_dashboard/courses/templates/courses/home.html

```diff
@@ -60,36 +54,4 @@
       </div>
     {% endfor %}
   </div>
-
```

Comment:
I don't think we actually want to remove this information, do we?  I'm wondering if Natalie just didn't include them in her mockup but didn't intend to remove them - they seem useful to me.